modules/src/dictionary.py

```diff
@@ -0,0 +1,18 @@
+import re
+import requests
+
+def match(input):
+	return bool(re.match(r'^.*\s+definition$', input))
+
+def process(input):
+	output = {}
+	word = re.match(r'^(?P<definition>.*)\s+definition$', input).group('definition')
+	try:
+		r = requests.get("https://glosbe.com/gapi/translate?from=eng&dest=eng&format=json&phrase=" + word + "&pretty=true")
+		data = r.json()
+		output['input'] = input
+		output['output'] = "Definition: " + data['tuc'][0]['meanings'][1]['text']
```

Comment:
Hmm, for cloud the 1 index gives the second definition, but for some words like 'flotsam', the 0 index simply just returns the word back. It seems as though the 1 index will always give some viable definition. Any suggestions though?